app/lib/pages/memory_detail/widgets.dart

```diff
@@ -206,7 +206,7 @@ class EventsListWidget extends StatelessWidget {
                   subtitle: Padding(
                     padding: const EdgeInsets.only(top: 4.0),
                     child: Text(
-                      '${dateTimeFormat('MMM d, yyyy', event.startsAt)} at ${dateTimeFormat('h:mm a', event.startsAt)} ~ ${minutesConversion(event.duration)}.',
+                      '${dateTimeFormat('MMM d, yyyy', utcToLocal(event.startsAt))} at ${dateTimeFormat('h:mm a', utcToLocal(event.startsAt))} ~ ${minutesConversion(event.duration)}',
```

Comment:
toLocal should be done at the schema parsing(fromJson) ? if not pls put the toLocal code there. just like this:

<img width="494" alt="Screenshot 2024-10-20 at 04 53 58" src="https://github.com/user-attachments/assets/2dae087f-1c35-4b73-b514-2a6a6bfd50c0">

Reply:
Oh yes totally forgot, let me do it